lib/tty.js

```diff
@@ -88,15 +89,16 @@ function WriteStream(fd) {
     throw new ERR_INVALID_FD(fd);
 
   const ctx = {};
-  const tty = new TTY(fd, false, ctx);
+  const tty = new TTY(fd, true, ctx);
```

Comment:
If you want, feel free to remove the parameter entirely, all the way down to the libuv `uv_tty_init()` call (it is ignored by libuv and can just be set to `0` always)